app/src/main/java/com/github/dedis/student20_pop/ui/IdentityFragment.java

```diff
@@ -0,0 +1,97 @@
+package com.github.dedis.student20_pop.ui;
+
+import android.graphics.Bitmap;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.ImageView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.github.dedis.student20_pop.OrganizerActivity;
+import com.github.dedis.student20_pop.PoPApplication;
+import com.github.dedis.student20_pop.R;
+import com.github.dedis.student20_pop.model.Keys;
+import com.github.dedis.student20_pop.utility.security.PrivateInfoStorage;
+
+import net.glxn.qrgen.android.QRCode;
+
+/**
+ * Represents the identity of a user within an organisation
+ * (which allows users to “wear different hats” in different organizations)
+ */
+public class IdentityFragment extends Fragment {
+    public static final String TAG = IdentityFragment.class.getSimpleName();
+
+    private EditText identityNameEditText;
+    private EditText identityTitleEditText;
+    private EditText identityOrganizationEditText;
+    private EditText identityEmailEditText;
+    private EditText identityPhoneEditText;
+    private ImageView qrCode;
+
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_identity, container, false);
+
+        // TODO :   The QR code does not appear at all unless the Name field is non-empty
+        //  and not all whitespace.
+        CheckBox anonymousCheckBox = view.findViewById(R.id.checkbox_anonymous);
+        qrCode = view.findViewById(R.id.qr_code);
+        identityEmailEditText = view.findViewById(R.id.identity_email);
+        identityNameEditText = view.findViewById(R.id.identity_name);
+        identityOrganizationEditText = view.findViewById(R.id.identity_organization);
+        identityPhoneEditText = view.findViewById(R.id.identity_phone);
+        identityTitleEditText = view.findViewById(R.id.identity_title);
```

Comment:
Who (other than the user) can see these when the user is not anonymous?

Reply:
For the moment, we do not use theses fields at all